articles/install-binder-qdk.md

```diff
@@ -13,16 +13,47 @@ uid: microsoft.quantum.install-qdk.overview.binder
+1. On the **Quantum Development Kit Samples** landing page, click the **Q# notebook** link on the **Intro to IQ#** sample to learn how to use IQ# and write your own quantum application notebooks.
 
-![QDK landing page](~/media/binder-install.png)
+![QDK samples overview - highlight Q# notebook link](~/media/binder-install.png)
+
+Note that you are not restricted to the existing samples, as you can create new notebook or text files by first selecting **File** -> **Open...** from the Jupyter interface to open the directory view, and then hitting the **New▾** button in the top right of the page.
+
+> [!WARNING]
+> Files created within the pre-configured binder are not persistent and should thus only be used for exploratory purposes.
+
+### Run Jupyter Notebook samples
+
+Binder supports both types of Q# development styles for Jupyter Notebook:
+
+- the *Q# notebook*, which uses the IQ# kernel to directly run code cells written in Q# (see [here](xref:microsoft.quantum.install-qdk.overview.jupyter)).
+- the *Q# + Python* notebook, which contains regular Python code that calls into Q# operations from a `.qs` file (see [here](xref:microsoft.quantum.install-qdk.overview.python)).
+
+You will find that the different Jupyter samples might use either of the two styles, and are sometimes available in both, so feel free to explore what best suites your preferences. You can create a notebook of your own by clicking on **New▾ → Python 3** (*Q# + Python* style) or **New▾ → Q#** (*Q# notebook* style) from the directory view.
+
+![Create a new notebook in Jupyter - highlight Python and Q# options](~/media/binder-new-notebook.png)
+
+### Run console application samples
+
+In addition to notebooks, you can also run Q# console applications via Binder. To do so, you'll have to open a terminal in the Jupyter interface, by selecting **New▾ → Terminal** from the directory view. You can then run bash commands, such as to run the CHSH sample as shown below.
+
+![Run Q# from a Jupyter terminal](~/media/binder-terminal.png)
+
+Here as well you might notice that there are two types of console applications present in the samples:
+
+- the *Q# standalone* application, which uses an `EntryPoint` function/operation in a `.qs` file to run the Q# program from the command line (see [here](xref:microsoft.quantum.install-qdk.overview.standalone)).
+- the *Q# + .NET* application, which uses a .NET language (C# or F#) host program to call into operations from a `.qs` file (see [here](xref:microsoft.quantum.install-qdk.overview.cs)).
+
+As an alternative to the Jupyter terminal, you can also run Q# console applications from a .NET PowerShell notebook such as the CHSH README.md shown below, but also from your own notebook via **New▾ → .NET (PowerShell)**.
+
+![Run Q# from a PowerShell notebook](~/media/binder-powershell-notebook.png)
 
 ## Use Binder with your own notebooks and repository
 
```

Comment:
As a note for the sentence on on line 60, it might make sense to clarify that this GitHub repository has to be public (Binder doesn't see private repos)

Reply:
Sounds good.